docs/source/system_reference_guide/oss_documentation/doi_and_licensing.ipynb

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
<p>Maybe convert this to a list for readability?</p><p> </p><p>Software should be made available in a "publicly accessible repository that is widely recognized by the community", should be "citable using a persistent identifier" (i.e. a DOI), and "should be released under a permissive license that has broad acceptance in the community".</p><p> </p><p>Is there a link for "policy documentation"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Made into a list and mentioned to refer to the SPD-41a. There are also some additional policies - should those be included? I added to the disclaimer and said there can be additional policies if NASA-funded, but figured licensing/doi were the main ones we talked about for MAAP users.</p>

Reply:
Or maybe mention all requirements for software, and have a section below that states if you're a MAAP user that isn't NASA-funded, we encourage the repository/doi/license?

Reply:
<p>I think the SPD-41a link is sufficient if that's the one we're referring to</p>